src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -392,6 +392,7 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 				switch result {
 				case .success:
 					Log.info("Analytics data succesfully submitted", log: .ppa)
+					Log.info("Analytics submission post-processing self-reference is nil: \(self == nil)", log: .ppa)
```

Comment:
we could add here a guard with a strong self reference, too. So we would be sure, that self is not nil and all the post-processing that depends on self will be executed correctly.

Reply:
If you can still get that into 2.10... otherwise, let's stick only with logging. According to @nickguendling, `self` shouldn't be `nil` because you are holding a strong reference elsewhere.
/fyi @marcussc

Reply:
Lets stick with the logging for now, so short before last RC.